on-prem-docs/logs.md

```diff
@@ -0,0 +1,62 @@
+## Logs
+
+## Logging
+
+### Logging Levels
+
+The recognized values for logging are: `error`, `warn`, `info`, `debug`, and `trace`.
+For a more detailed explanation of logging in Chef Habitat, see the [Supervisor Log Configuration Reference](https://www.habitat.sh/docs/reference/#supervisor-log-configuration-reference) and the [Supervisor Log Key](https://www.habitat.sh/docs/reference/#supervisor-log-key) documentation.
+
+### Basic Logging
+
+To turn on and examine the services debug logging in your Habitat installation:
+
+1. Edit the `sudo /hab/svc/builder-api/user.toml` file
+1. On the first line, change the log_level from **error** to **debug**
+
+    ```toml
+    log_level="debug,tokio_core=error,tokio_reactor=error,zmq=error,hyper=error"
+    ```
+
+1. Save and close the file
+1. Restart Habitat with `sudo systemctl restart hab-sup` to restart the habitat.
+1. Use `journalctl -fu hab-sup` to view the logs.
+1. Reset `/hab/svc/builder-api/user.toml` file to the default `log_level=error` and restart the services with `sudo systemctl restart hab-sup` to restore error-level logging.
+
+### RUST_LOG
+
+1. Use `RUST_LOG=debug RUST_BACKTRACE=1` to see an individual command's debug and backtrace.
+
+    ```bash
+    # Linux/MacOS
+    # replace "hab sup run" with your command
+    RUST_LOG=debug RUST_BACKTRACE=1 hab sup run
+    ```
+
+1. Edit the `sudo /hab/svc/builder-api/user.toml` file
+1. On the second line, change:
+
+    ```toml
+    RUST_LOG=debug RUST_BACKTRACE=1
+    ```
```

Comment:
This doesn't show what you're changing things from/to.  If this is just an addition, perhaps change line 37 from `change:` to `add:`.  Otherwise, it'd be helpful to provide more context around what's changing.